apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.tsx

```diff
@@ -176,6 +176,7 @@ export function AssessmentQuestion({
             rubricData={rubric_data}
             csrfToken={__csrf_token}
             aiGradingStats={aiGradingStats}
+            resLocals={resLocals}
```

Comment:
We can't pass in `resLocals` to hydrated components for security reasons. You should be able to extract what you want with the `page-context` and `getAssessmentContext` helpers. We may need a new `getQuestionContext` helper.